package.json

```diff
@@ -91,19 +91,15 @@
     "@netlify/eslint-config-node": "^4.1.2",
     "ava": "^3.0.0",
     "babel-loader": "^8.2.2",
+    "c8": "^7.11.0",
     "cpy": "^8.1.0",
     "cpy-cli": "^3.1.1",
     "cypress": "^7.0.0",
     "del": "^6.0.0",
-    "eslint-plugin-cypress": "^2.11.2",
```

Comment:
These are included in `@netlify/eslint-config-node`

Reply:
Good catch!